components/esp_lvgl_port/include/esp_lvgl_port.h

```diff
@@ -20,6 +20,10 @@
 #include "esp_lcd_touch.h"
 #endif
 
+#if __has_include ("iot_button.h")
```

Comment:
Since you have `button` component in the dependencies list of esp_lvgl_port, when can this condition be false?

Reply:
As we solved it in another thread, this should left here.